### PR TITLE
add subscriptionData to ResolveTokenResult

### PR DESCRIPTION
**Pull request by romanbracinik**
Slo by tam prosim este pridat `subscriptionData`

### CODE REVIEW

File: src/Model/ResolveTokenResult.php [hunk 1]
```diff
@@ -17,6 +18,7 @@ public static function fromResponse(array $data): self
     {
         return new self(
             $data['subscriptionId'],
+            $data['subscriptionData'],
```
Comment (romanbracinik) [inline src/Model/ResolveTokenResult.php:21]: Potreboval by som este nejake data naviac co pouzivame pre azure, povodne sme mali toho klienta c&p v connection https://github.com/keboola/connection/pull/4772/files#diff-3e219885e47031312bdd6aa5508cb972f58a7a0d0ffaa42f422716476d0b9ea9L33

Comment (pepamartinec) [reply src/Model/ResolveTokenResult.php:21]: Zajimavy, ze to tam pred tim nebylo :thinking: Nevis jak to fungovalo??

Comment (pepamartinec) [reply src/Model/ResolveTokenResult.php:21]: Aha, vy jste v Connection meli nejakeho vlastniho Billing API clienta, ze?

**Review by pepamartinec — APPROVED** [review]
lgtm

File: src/Model/ResolveTokenResult.php (hunk 1 above)
Comment (romanbracinik) [reply src/Model/ResolveTokenResult.php:21]: Jj bol to c&p, neviem ci to Tomas vtedy odniekal vykopiroval a pridal tom tam alebo to tam niekde bolo

**Pull request merged**